src/components/cluster/Cluster.tsx

```diff
@@ -231,7 +234,10 @@ function Cluster({
     const [config, setConfig] = useState(defaultConfig)
     const [prometheusAuth, setPrometheusAuth] = useState(undefined)
     const [showClusterComponentModal, toggleClusterComponentModal] = useState(false)
-    const [, grafanaModuleStatus, ] = useAsync(() => getModuleInfo(ModuleNameMap.GRAFANA), [clusterId])
+    const [, grafanaModuleStatus] = useAsync(
+        window._env_.K8S_CLIENT ? null : () => getModuleInfo(ModuleNameMap.GRAFANA),
+        [clusterId],
+    )
```

Comment:
Please make use of 3rd arg `shouldRun` here. Pass `!window._env_.K8S_CLIENT` to it so that it'll be executed only when K8S_CLIENT is false.

Reply:
Done